packages/core/src/index.ts

```diff
@@ -147,7 +151,10 @@ export type HoprOptions = {
   allowPrivateConnections: boolean
   password: string
   connector?: HoprCoreEthereum
-  strategy: ChannelStrategyInterface
+  strategy?: {
+    name: Strategy
+    settings?: any
+  }
```

Comment:
This will force a change in the config revamp PR.

Reply:
Unfortunately it was not possible to keep it like this with the new Strategy setup.
But I think it is better to keep a config object in `HoprOptions` than an already instantiated Strategy, and instantiate it later.

Reply:
There's no `HoprOptions` in the revamped config.